### DISCUSSION TITLE
Duda del alcance de la iteracion

### DISCUSSION THREAD

**Discussion by francorojo** [dds-jv/foro#129] [TPA - 2020] [answered]
Buenaas, tengo una duda sobre el alcance en esta iteracion, cuando dice:  "Como usuarie de QuéMePongo, quiero poder recibir sugerencias de atuendos que tengan una prenda para cada categoría, aunque a futuro podrán tener más", dejamos armada la estructura para poder recibir estos conjuntos, o llego el momento de hacerlos? porque yo habia pensado en la primera iteracion una forma de hacerlos, me costo mucho pensar y me salio, en teoria, muy a lo "c" con 4 for por cada lista de prendas (seguramente exista alguna funcion de orden superior que se adapte al caso). La agrego ahora o todavia no?

Y la temperatura de cada prenda hasta la cual es adecuada, podria ser un atributo que por el momento la podriamos setear nosotros? pero q seguramente en el futuro el usuario o alguien se encargara de determinar?

**Comment by roli93** [comment]
@francorojo Creo que acabo de responder esto en el mail de recién. Puede ser?

**Comment by francorojo** [comment]
Perfectooo, muchas graciaaas!

**Comment by roli93** [comment]
@francorojo Bueno no sé por qué esto que me llego por mail no aparece acá:

> Buenaaas de vuelta perdon, una consulta, para poder recibir las sugerencias de prendas, en este caso conviene mas crear una clase GeneradorDePrendas a la cual poder preguntarle, o ya crear la clase Negocio/QueMePongo/Aplicacion que me devuelva la sugerencia como un metodo?

Recordá una vez q el issue está cerrado siemrpe abrir uno nuevo. Respondiendo a tu pregunta, mirate los ultimos mensajes de este issue: #25

**Comment by francorojo** [comment, marked as answer]
Perfectooo, muchas graciaaaas!
